.golangci.yml

```diff
@@ -112,13 +112,8 @@ linters-settings:
     # Forbid any `replace` directives that are intended temporarily only during
     # development. The modules listed below are intended to be replaced permanently.
     replace-allow-list:
-      - github.com/Microsoft/go-winio
       - github.com/Shopify/sarama
-      - github.com/dop251/goja
-      - github.com/dop251/goja_nodejs
-      - github.com/fsnotify/fsnotify
```

Comment:
I believe these will still be needed for enabling the filebeat receiver - https://github.com/elastic/elastic-agent/pull/5666

Reply:
Thanks.  Added them back in bbefe5e9bf.